### DISCUSSION TITLE
Add iOS live activities

### DISCUSSION THREAD

**Discussion by astashov** [astashov/liftosaur#135] [Ideas]
Could show a running timer there, maybe a current exercise with completed/non-completed sets/reps/weights.

**Comment by StanleyMitchell8** [comment]
I would love this, I also feel like this would be a good addition while working on the watch integration.  It would speed up the interaction on the phone. Another app i was using has both and they are very nice additions.  Love your app btw and looking forward to your improvements.

**Comment by astashov** [comment]
Done for both iOS and Android
